Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.yml

```diff
@@ -1088,11 +1088,181 @@ script:
+      description: Indicates whether the scan history is archived or not.
+      type: Boolean
+    - contextPath: TenableIO.ScanHistory.time_start
+      description: The start time of the scan.
+      type: Number
+    - contextPath: TenableIO.ScanHistory.visibility
+      description: The visibility of the scan.
+      type: String
+    - contextPath: TenableIO.ScanHistory.targets.custom
+      description: Indicates whether custom targets were used in the scan.
+      type: Boolean
+    - contextPath: TenableIO.ScanHistory.targets.default
+      description: Indicates whether the default targets were used in the scan.
+      type: Boolean
+    - contextPath: TenableIO.ScanHistory.status
+      description: The status of the scan.
+      type: String
+  - arguments:
+    - name: scanId
+      description: The identifier for the scan to export. Run the "tenable-io-list-scans" command to get all available scans.
+      required: true
+    - name: historyId
+      description: The unique identifier of the historical data to export. Run the "tenable-io-get-scan-history" command to get history IDs.
+    - name: historyUuid
+      description: The UUID of the historical data to export. Run the "tenable-io-get-scan-history" command to get history UUIDs.
+    - name: format
+      description: |
+        The file format to export the scan in. Scans can be export in the HTML and PDF formats for up to 60 days.
+         For scans that are older than 60 days, only the Nessus and CSV formats are supported. Default is CSV.
+      required: true
+      defaultValue: CSV
+      auto: PREDEFINED
+      predefined:
+      - Nessus
+      - HTML
+      - PDF
+      - CSV
+    - name: chapters
+      description: |
+        A list of chapters to include in the export.
+         Chapters can be any of the following options: vuln_hosts_summary, vuln_by_host, compliance_exec, remediations, vuln_by_plugin, compliance.
+         This argument is required if the file format is PDF or HTML.
+      isArray: true
+      auto: PREDEFINED
+      predefined:
+      - vuln_hosts_summary
+      - vuln_by_host
+      - compliance_exec
+      - remediations
+      - vuln_by_plugin
+      - compliance
+    - name: filterName
+      description: |
+        A list of filters to apply to the exported scan report.
+         Run the "tenable-io-list-scan-filters" command to get filter names ("Filter name" in response).
+         The values of the "filterName", "filterQuality" and "filterValue" lists are matched sequentially to produce individual filters.
+      isArray: true
+    - name: filterQuality
+      description: |
+        A list of operators for the filter to apply to the exported scan report.
+         Run the "tenable-io-list-scan-filters" command to get filter qualities ("Filter operators" in response).
+         The values of the "filterName", "filterQuality" and "filterValue" lists are matched sequentially to produce individual filters.
```

Comment:
lets give an example of how to use the parallel arrays here in the description of the command, and explain how what gets paired with what